index.js

```diff
@@ -284,9 +287,12 @@ class ThreadStream extends EventEmitter {
     })
   }
 
-  flushSync () {
+  flushSync (cb) {
     if (this[kImpl].destroyed) {
-      throw new Error('the worker has exited')
+      if (typeof cb === 'function') {
+        process.nextTick(cb, new Error('the worker has exited'))
+      }
```

Comment:
flushSync is synchronous, so let's not have a callback. let's just silently return.